hal/src/rtl872x/gpio_hal.cpp

```diff
@@ -159,6 +159,7 @@ int hal_gpio_configure(hal_pin_t pin, const hal_gpio_config_t* conf, void* reser
         }
 
         Pinmux_Config(hal_pin_to_rtl_pin(pin), PINMUX_FUNCTION_GPIO);
+        hal_gpio_configure_drive_strength(pin, HAL_GPIO_DRIVE_DEFAULT);

```

Comment:
The `hal_gpio_config_t* conf` parameter here has a field for `drive_strength`. Should we use that field instead of the default? 
(I believe the NRF MCU had GPIO settings for drive strength as well, so perhaps we can re-use this field for the RTK platform now)

See `hal_gpio_mode()` for the config struct, and perhaps we initialize `conf.drive_strength = HAL_GPIO_DRIVE_DEFAULT` there?

This should also implement `pinSetDriveStrength()` wiring API function as well.

Reply:
👍🏻 Fixed, the `pinSetDriveStrength()` function is already available for the Realtek platform.